patches/build-config-chrome_build.gni.patch

```diff
@@ -1,20 +0,0 @@
-diff --git a/build/config/chrome_build.gni b/build/config/chrome_build.gni
-index e8f9660f2c3d0def9307e4ebd2fd3e293150e6a7..ba5b9c51706e72e6fce3a240645303d5d61b02c6 100644
---- a/build/config/chrome_build.gni
-+++ b/build/config/chrome_build.gni
-@@ -1,6 +1,7 @@
- # Copyright 2015 The Chromium Authors. All rights reserved.
- # Use of this source code is governed by a BSD-style license that can be
- # found in the LICENSE file.
-+import("//brave/build/config/brave_build.gni")
```

Comment:
If removing this line from that patch causes all the new patches below, then I think we'd want to keep this line and just remove the branding line below.

Reply:
I had exactly the same thoughts while doing this, but then I figured that since the lines added everywhere else go into patches that already existed, then perhaps it did make some sense in the end, since we would be getting rid of one patch file after all. No strong opinion, though, looking forward to seeing what others think.

Reply:
@mkarolin @bridiver Hi again! Sorry for pushing, but since this PR has sit here for a few days already and it's blocking https://github.com/brave/brave-browser/pull/10758 as well, I believe a ping is in order.

@bridiver Could you clarify whether you'd be ok with this change even if it implies modifying several other -already existing- patches? (yet it allows us to drop one, see my comment above)

Thanks in advance!

Reply:
@mariospr we're not trying to minimize the number of files with patches, we are trying to minimize the total number of patches and therefore seems to be moving in the wrong direction. I agree with @mkarolin that it doesn't make sense to remove this here only to have to add it to a bunch of other places. We specifically added it here so it would be available pretty much everywhere.